datasets/officehome.py

```diff
@@ -0,0 +1,97 @@
+from benchopt import BaseDataset, safe_import_context
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from sklearn.preprocessing import LabelEncoder
+    from skada.utils import source_target_merge
+    import torchvision.transforms as transforms
+    from torch.utils.data import DataLoader
+    from benchmark_utils.utils import download_and_extract_zipfile, ImageDataset
+
+
+# All datasets must be named `Dataset` and inherit from `BaseDataset`
+class Dataset(BaseDataset):
+
+    # Name to select the dataset in the CLI and to display the results.
+    name = "OfficeHome"
+
+    # List of parameters to generate the datasets. The benchmark will consider
+    # the cross product for each key in the dictionary.
+    # Any parameters 'param' defined here is available as `self.param`.
+    parameters = {
+        'source_target': [
+            ('art', 'clipart'),
+            ('art', 'product'),
+            ('art', 'realworld'),
+            ('clipart', 'art'),
+            ('clipart', 'product'),
+            ('clipart', 'realworld'),
+            ('product', 'art'),
+            ('product', 'clipart'),
+            ('product', 'realworld'),
+            ('realworld', 'art'),
+            ('realworld', 'clipart'),
+            ('realworld', 'product'),
+        ],
+    }
+
+    path_dataset = "data/OfficeHome.zip"
+    path_extract = "data/OfficeHome/"
+    url_dataset = "https://wjdcloud.blob.core.windows.net/dataset/OfficeHome.zip"
+
+    def _get_dataset(self, domain_select):
+        # Define transformations to preprocess the images
+        preprocess = transforms.Compose([
+            transforms.Resize(256),
+            transforms.CenterCrop(224),
+            transforms.ToTensor(),
+            transforms.Normalize(
+                mean=[0.485, 0.456, 0.406],
+                std=[0.229, 0.224, 0.225],
+            ),
+        ])
```

Comment:
where did you get these values?

Reply:
The values are the ones used to train a resnet on imagenet. Then, from https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html we should use these values for transfer learning.